api/client/utils_test.go

```diff
@@ -32,7 +32,7 @@ func defaultServerConfig(t *testing.T) server.Config {
 	ipfsAddr := util.MustParseAddr(ipfsAddrStr)
 
 	devnet := tests.LaunchDevnetDocker(t, 1, 300, ipfsAddrStr, false)
-	devnetAddr := util.MustParseAddr("/ip4/127.0.0.1/tcp/" + devnet.GetPort("7777/tcp"))
+	devnetAddr := "127.0.0.1/tcp/" + devnet.GetPort("7777/tcp")
```

Comment:
I guess this should be:
`devnetAddr := "127.0.0.1:" + devnet.GetPort("7777/tcp")`

Reply:
👍🏼